the-internet/pom.xml

```diff
@@ -31,9 +31,9 @@
 			<version>7.1.0</version>
 			<scope>test</scope>
 		</dependency>
-		
+
 		<!-- json-simple -->
```

Comment:
BrowserFactory:
The responsibility of this class is to determine which browserManager to use. No settings for the webDrivers should be determined here. See the UML diagram in the Powerpoint slide over Design Patterns.